conda/Dockerfile

```
@@ -4,4 +4,5 @@ ARG ROOT_VERSION=6.20.00
 ARG PYTHON_VERSION=3.7
 
 RUN conda config --add channels conda-forge && \
-    conda install --quiet python=$PYTHON_VERSION root=$ROOT_VERSION
+    conda install --quiet python=$PYTHON_VERSION root=$ROOT_VERSION && \
+    conda clean --all
```

Using this takes squeezes another 100MB out and takes the image from 1.7GB to 933MB:
```suggestion
    conda config --set allow_softlinks false && \
    conda config --set always_copy true && \
    conda install --yes --quiet python=$PYTHON_VERSION root=$ROOT_VERSION && \
    conda clean --yes --all --force-pkgs-dirs
```